src/libsync/creds/httpcredentials.h

```diff
@@ -171,6 +171,8 @@ private Q_SLOTS:
     bool _keychainMigration = false;
     bool _retryOnKeyChainError = true; // true if we haven't done yet any reading from keychain
 
+    bool _useOauth = false;
```

Comment:
I don't understand how this var is used.

When is it true, when false?

Reply:
Sorry that was bullshit, it should be more sane now.